addons/mail/static/src/core/common/autoresize_input.js

```diff
@@ -32,8 +34,19 @@ export class AutoresizeInput extends Component {
                 this.state.value = nextProps.value;
             }
         });
-        onExternalClick("input", () => this.onValidate());
+        onExternalClick("input", async (ev, { downTarget }) => {
+            if (downTarget === this.inputRef.el) {
+                return;
+            }
+            this.onValidate();
+        });
```

Comment:
This fixes renaming in chat window ending when selection triggers external click.
External click whose related mousedown target is inside input do not ends the renaming.